cli/lib/appdata.ts

```diff
@@ -33,6 +33,7 @@ const userDataSchema = z
 			} )
 			.passthrough()
 			.optional(),
+		lastBumpStats: z.record( z.string(), z.record( z.string(), z.number() ) ).optional(),
```

Comment:
```suggestion
		lastBumpStats: z
			.record( z.nativeEnum( StatsGroup ), z.record( z.nativeEnum( StatsMetric ), z.number() ) )
			.optional(),
```
If we use a single `StatsGroup` and `StatsMetric` definition, we could refine the schema.